ops/services/app_service/metabase-readonly/vault.tf

```diff
@@ -0,0 +1,15 @@
+resource "azurerm_key_vault_secret" "postgres_readonly_user" {
+  key_vault_id = var.global_vault_id
+  name         = "simple-report-${var.env}-postgres_readonly_user"
+  value        = "readonly"
+}
+
+data "azurerm_key_vault_secret" "postgres_readonly_pass" {
+  name         = "simple-report-${var.env}-postgres_readonly_pass"
+  key_vault_id = var.global_vault_id
+}
+
+# We could also set this as a random password, as is done in
```

Comment:
I think relying on a random password and Vault to do this is the better approach, since it's one less secret to manage and configure.

Reply:
Changed this to a random password, though I'm still not sure I'm setting up the key vault secrets correctly.